CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+
+* We are using following labels for issue work-flow:
+   - **Backlog** - if issues has **not been planned for current release cycle**
+   - **Release blocker** - if issue is **blocking the release**
+   - **Priority: high** - issues with this label **should be resolved as quickly as possible**
+   - **Priority: low** - issues with this label **won’t have the immediate focus of the core team**
+
+**If you want to introduce a new label then you need to raise a PR to update this document with the new label details.**
+
+## Steps to Contribute
+velero-plugin is an Apache 2.0 Licensed project and all your commits should be signed with Developer Certificate of Origin. See [Sign your work](#sign-your-work).
+
+For setting up a Development environment on your local host, see the detailed instructions [here](developer-setup.md).
+
+* Find an issue to work on or create a new issue. The issues are maintained at [openebs/velero-plugin](https://github.com/openebs/velero-plugin/issues). You can pick up from a list of [good-first-issues](https://github.com/openebs/velero-plugin/labels/good%20first%20issue).
+* Claim your issue by commenting your intent to work on it to avoid duplication of efforts.
+* Fork the repository on GitHub.
+* Create a branch from where you want to base your work (usually master).
+* Commit your changes by making sure the commit messages convey the need and notes about the commit.
+* Please make sure than your code is aligned with the standard mentioned at [code-standard](code-standard.md).
+* Verify that your changes passes `make lint` or `make lint-docker` (docker version of `make lint`)
+* Push your changes to the branch in your fork of the repository.
+* Submit a pull request to the original repository. See [Pull Request checklist](#pull-request-checklist)
+
+## Pull Request Checklist
+* Rebase to the current master branch before submitting your pull request.
+* Commits should be as small as possible. Each commit should follow the checklist below:
+  - For code changes, add tests relevant to the fixed bug or new feature.
+  - Commit header (first line) should convey what changed
+  - Commit body should include details such as why the changes are required and how the proposed changes help
+  - DCO Signed, please refer [signing commit](code-standard.md/sign-your-commits)
+* If your PR is about fixing a issue or new feature, make sure you add a change-log. Refer [Adding a Change log](code-standard.md/adding-a-changelog)
+* PR title must follow convention: `<type>(<scope>): <subject>`.
+
+  For example:
+  ```
+   feat(backup): support for backup to aws
+   ^--^ ^-----^   ^-----------------------^
+     |     |         |
+     |     |         +-> PR subject, summary of the changes
+     |     |
+     |     +-> scope of the PR, i.e. component of the project this PR is intend to update
+     |
+     +-> type of the PR.
+  ```
+
+   `type` is defining if release will be triggering after merging submitted changes
```

Comment:
Where is this defined? and how is the release triggered?

Reply:
this line is not required. need to be removed.